gem/lib/earlgrey/cli.rb

```diff
@@ -0,0 +1,50 @@
+#
+#  Copyright 2016 Google Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+module EarlGrey
+  class CLI < Thor
+    package_name 'EarlGrey'
+
+    desc 'install', 'Installs EarlGrey into an Xcode unit test target'
+    method_option :project, aliases: '-p', type: :string, required: false, desc: 'Project'
+    method_option :target,  aliases: '-t', type: :string, required: true, desc: 'EarlGrey'
```

Comment:
Cocoapods won't be going through the CLI flow as the `configure_for_earlgrey` method is invoked directly. The default there is already false since swift/carthage are never set.

For CLI, the only point to using it is for carthage support so I'm not sure why we'd want carthage to be false?